CartoDB/cartodb
# Store all attributes changes when a feature is edited until it is saved

It will fix https://github.com/CartoDB/cartodb/issues/11010. Basically we store all attributes changed when it is edited before saving it. After that, changes history should be empty. 

**STTest**
1. Create a map of lines|points|polygons (take a dataset with several columns with data please).
2. Try to edit a feature.
3. Change the geometry (move the point, or any vertex of the line|polygon).
4. Edit any field of the feature form.
5. Save the changes.
6. Map should be refreshed.

cc @matallo
